src/test/scala/uk/gov/hmrc/test/api/client/WsClient.scala

```diff
@@ -0,0 +1,81 @@
+package uk.gov.hmrc.test.api.client
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import akka.util.ByteString
+import com.typesafe.scalalogging.LazyLogging
+import play.api.libs.json.{JsNull, JsValue, Json}
+import play.api.libs.ws._
+import play.api.libs.ws.ahc.StandaloneAhcWSClient
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+object WsClient extends LazyLogging {
+  val timeout: FiniteDuration = 60 seconds
+
+  implicit val bodyWrites: BodyWritable[JsValue] = {
+    BodyWritable(a => InMemoryBody(ByteString.fromArrayUnsafe(Json.toBytes(a))), "application/json")
+  }
+  private val asyncClient: StandaloneAhcWSClient = {
+      implicit val system:       ActorSystem       = ActorSystem()
+      implicit val materializer: ActorMaterializer = ActorMaterializer()
+
+    StandaloneAhcWSClient()
+  }
+
+  def get(uri: String): StandaloneWSResponse = {
```

Comment:
Would it be possible to call the other get method from here to avoid code duplication?
The call to Await.result() can be written by adding if condition to check if the other parameters are null.

Reply:
Have removed the single param get method. Remaining method works with no header, queryparameters etc. Happy to consider any other alternatives